aim/sdk/tracker.py

```diff
@@ -116,23 +116,18 @@ def _track(
         if context is None:
             context = {}
 
-        if is_number(value):
-            val = convert_to_py_number(value)
-        elif isinstance(value, (CustomObject, list, tuple)):
-            val = value
-        else:
-            raise ValueError(f'Input metric of type {type(value)} is neither python number nor AimObject')
-
+        values = self._normalized_values(value, name)
         with self.repo.atomic_track(self.hash):
             ctx = Context(context)
-            seq_info = self.sequence_infos[ctx.idx, name]
-            if not seq_info.initialized:
-                self._init_sequence_info(ctx.idx, name, val)
-            step = step if step is not None else seq_info.count
+            for name, val in values.items():
+                seq_info = self.sequence_infos[ctx.idx, name]
+                if not seq_info.initialized:
+                    self._init_sequence_info(ctx.idx, name, val)
+                step = step if step is not None else seq_info.count
```

Comment:
doesn't this logic need to be changed, if dictionary is provided?

Reply:
nope. all the operations, including getting sequence info, updating, etc. are done on the level of dict item, which is the same as it was before.
so if you track a dict of 5 items, 5 sequence infos should be checked.